Symfony/CS/Fixer/Contrib/AlignDoubleArrowFixer.php

```diff
@@ -139,7 +139,7 @@ private function injectAlignmentPlaceholders(Tokens $tokens, $startAt, $endAt)
                 continue;
             }
 
-            if ($token->equals(',')) {
+            if ($token->equals(',') && !in_array($tokens[$index - 1]->getContent(), array(')', ']'), true)) {
```

Comment:
In this case you'll match `$token->isGivenKind(T_DOUBLE_ARROW)` part and the method will be called again.

Which test exactly you want me to add?